lib/exceptions.rb

```diff
@@ -64,7 +64,7 @@ def to_s
         end.compact.first || "<HOSTNAME>"
         formatted_scopes = Machinery::Ui.internal_scope_list_to_string(@scopes)
 
-        cmd = "#{$0} inspect --extract-files --scope=#{formatted_scopes}"
+        cmd = "#{$0} inspect --extract-files --scope=#{formatted_scopes.delete(' ')}"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.